README.md

```diff
@@ -1,4 +1,262 @@
+IPFS, libp2p or the RFC process itself. What constitutes a
+"substantial" change is evolving based on community norms and varies depending
+on what part of the ecosystem you are proposing to change, but may include the
+following.
+
+  - Any semantic or syntactic change to the protocols that is not a bugfix.
+  - Removing protocol features, including those that are feature-gated.
+  - Adding anything to our list of officially supported projects, libraries, and specifications.
+
+Some changes do not require an RFC:
+
+  - Rephrasing, reorganizing, refactoring, or otherwise "changing shape does not change meaning".
+  - Additions that strictly improve objective, numerical quality criteria
+    (warning removal, speedup, better platform coverage, more parallelism, trap more errors, etc.)
+  - Additions only likely to be _noticed by_ other developers-of-ipfs, but
+    invisible to users-of-ipfs.
+
+If you submit a pull request to implement a new feature without going through
+the RFC process, it may be closed with a polite request to submit an RFC first.
+
+
+### working group specific guidelines
+[working group specific guidelines]: #working group-specific-guidelines
+
+As working groups take form, they will have the prerogative to add further guidelines regarding RFCs that apply to their domain of work.
+
+## Before creating an RFC
+[Before creating an RFC]: #before-creating-an-rfc
+
+A hastily-proposed RFC can hurt its chances of acceptance. Low quality
+proposals, proposals for previously-rejected features, or those that don't fit
+into the near-term roadmap, may be quickly rejected, which can be demotivating
+for the unprepared contributor. Laying some groundwork ahead of the RFC can
+make the process smoother.
+
+Although there is no single way to prepare for submitting an RFC, it is
+generally a good idea to pursue feedback from other project developers
+beforehand, to ascertain that the RFC may be desirable; having a consistent
+impact on the project requires concerted effort toward consensus-building.
+
+The most common preparations for writing and submitting an RFC include talking
+the idea over on the #ipfs irc channel, filing and discussing ideas on the
+[RFC issue tracker], and occasionally posting "pre-RFCs" on the
+[ipfs discussion forum](https://discuss.ipfs.io) for early review.
+
+As a rule of thumb, receiving encouraging feedback from long-standing project
+developers, and particularly members of the relevant [working group] is a good
+indication that the RFC is worth pursuing.
+
+
+## The RFC life-cycle
+[The RFC life-cycle]: #the-rfc-life-cycle
+
+In short, to get a major feature added to IPFS, one must first get the RFC
+merged into the RFC repository as a markdown file. At that point the RFC is
+"Accepted" and may be implemented with the goal of eventual inclusion into the corresponding protocols or libraries.
+
+As it grows from an idea to shipping in a supported release, an SDK RFC will traverse along:
```

Comment:
This section mentions "SDK" a lot as it was copied from the Couchbase one. It should probably just sat "RFC"

I also asked for permission to copy & paste this section, it was granted. Could give credit/thanks/whatever in the readme to the Couchbase SDK one, I think that would be fair.

Reply:
could I get a link for citing the Couchbase source?

Reply:
https://github.com/couchbaselabs/sdk-rfcs :)